styles/styles.css

```diff
@@ -45,6 +54,7 @@ body {
 
 /*Шапка*/
 .page-header {
+  height: 66px;
```

Comment:
```suggestion
  min-height: 66px;
```
старайся не принимать ничего фиксированного